.php_cs

```diff
@@ -12,42 +12,17 @@ EOF;
 $config = PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
```

Comment:
We need to obey the cs rules defined here, *in this file, itself*. :-) Change to long array syntax here, and elsewhere in the file where short array syntax is used.